samples/tab-app-monetization/nodejs/ClientApp/package.json

```diff
@@ -3,26 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fluentui/react-northstar": "^0.58.0",
-    "@microsoft/teams-js": "^2.0.0",
+    "@fluentui/react-components": "^9.0.0-rc.10",
+    "@microsoft/teams-js": "^2.5.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "bootstrap": "^5.2.0",
     "msteams-ui-components-react": "^0.8.4",
-    "react": "^17.0.2",
     "react-bootstrap": "^2.4.0",
-    "react-dom": "^17.0.2",
     "react-json-view": "^1.21.3",
     "react-meta-tags": "^1.0.1",
-    "react-router-dom": "^5.3.0",
-    "react-scripts": "4.0.3",
-    "typescript": "^4.4.2",
+    "typescript": "^4.9.3",
     "typestyle": "^2.1.0",
-    "web-vitals": "^1.1.2"
+    "web-vitals": "^1.1.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.4.3",
+    "react-scripts": "^5.0.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set PORT=3978 && react-scripts --openssl-legacy-provider start",
```

Comment:
What is the benefit of it?
set PORT=3978 && react-scripts --openssl-legacy-provider start

Reply:
for it's we are using 3978 port.